unlock-protocol-com/blog/base.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Unlock Protocol Has Been Deployed to the Base L2 🔵
+authorName: Julien Genestoux
+publishDate: August 9, 2023
+description: You can use any of the tools available in the Unlock Protocol ecosystem to create membership smart contracts or ticket events on Base.
+image: https://docs.base.org/img/logo.svg
```

Comment:
@julien51 hotlinking to external image risks breaking if/when they change that image (probably not the case here, but have also seen folks block external references to images). safer to refer to an image on our domain.

Reply:
Good point1 Let me update that!